hlxsites/merative2
# fix(container): Adjust grid to align with design + adjust button hover effects

## Issue

Fixes - https://jira.sdlc.merative.com/browse/MERATIVE-787

## Description

**Changed**

- This PR adds max-width value to the section/container so that it's fluid instead of having fixed width of each breakpoint that we currently have right now.
- In addition, this PR fixes design QA issues seen on the current version of the homepage.
- Adds easing effect to button styles
- Overall, this PR addresses some of the design QA issues that did not get addressed when we launched the version of the homepage in Franklin.

## Design Specs

- Figma Link - https://www.figma.com/file/FSqQ1LYrVSssEFOZuOnMIE/Final-page-layouts-for-Merative.com?type=design&node-id=202%3A40370&mode=design&t=0pO1gDaVFwT9c9qo-1 

## Test URLs

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/
- After (Changes from this PR): https://fix-grid-container--merative2--proeung.hlx.page/

https://github.com/hlxsites/merative2/assets/1815714/e0e74ae1-deed-4921-929c-27f3c9333208

## Testing Instruction

- Visit the homepage (https://fix-grid-container--merative2--proeung.hlx.page/) and pages that are routed over to Franklin (see - https://confluence.sdlc.merative.com/display/MERATIVE/CDN+Routing+Rules)
- Ensure that the container is fluid and matches the design specs.
